recipes/openjdk/bld.bat

```diff
@@ -0,0 +1,11 @@
+MOVE bin\* %LIBRARY_BIN%
+MOVE include\* %LIBRARY_INC%
+MOVE jre %LIBRARY_PREFIX%\jre
+MOVE lib\* %LIBRARY_LIB%
+
+:: ensure that JAVA_HOME is set correctly
+for %%x in (activate deactivate) do (
+    set "dir=%PREFIX%\etc\conda\%%x.d"
+    mkdir "!dir!"
+    copy "%RECIPE_DIR%\scripts\%%x.bat" "!dir!"
```

Comment:
It doesn't; the scripts are going into the etc\conda\activate.d folder.  These scripts are the ones he has below.  They are the java_home.sh equivalents, but by renaming them, it is much less clear what they are.

Maybe you were trying to clean up the implementation of setting the environment variables from the Continuum recipe (https://github.com/ContinuumIO/anaconda-recipes/pull/65/files#diff-fc82b4981d50306fdcf15e1b27dbdb54R8 ).  This loop really doesn't save that much code, but it does make things more complicated.

Please, just use the Continuum recipe's code here, unless you see something wrong with it.